html-css/portfolio/script.js

```diff
@@ -0,0 +1,192 @@
+            easing: "easeInExpo",
+            duration: 4100,
+            delay: (el, i) => 100 + 30 * i
+        });
+}
+
+function applyML12ToAll() {
+    const elements = document.querySelectorAll('.ml12');
+    elements.forEach(element => {
+        applyML12Animation(element);
+    });
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+    applyML12ToAll();
+});
+
+
+// ML 11
+var textWrapper = document.querySelector('.ml11 .letters');
+textWrapper.innerHTML = textWrapper.textContent.replace(/([^\x00-\x80]|\w)/g, "<span class='letter'>$&</span>");
+
+anime.timeline({loop: true})
+  .add({
+    targets: '.ml11 .line',
+    scaleY: [0,1],
+    opacity: [0.5,1],
+    easing: "easeOutExpo",
+    duration: 700
+  })
+  .add({
+    targets: '.ml11 .line',
+    translateX: [0, document.querySelector('.ml11 .letters').getBoundingClientRect().width + 10],
+    easing: "easeOutExpo",
+    duration: 700,
+    delay: 100
+  }).add({
+    targets: '.ml11 .letter',
+    opacity: [0,1],
+    easing: "easeOutExpo",
+    duration: 2600,
+    offset: '-=775',
+    delay: (el, i) => 34 * (i+1)
+  }).add({
+    targets: '.ml11',
+    opacity: 0,
+    duration: 1000,
+    easing: "easeOutExpo",
+    delay: 1000
+  });
+
+//ML17
+
+
+var textWrapper = document.querySelector('.ml7 .letters');
+textWrapper.innerHTML = textWrapper.textContent.replace(/\S/g, "<span class='letter'>$&</span>");
+
+anime.timeline({loop: true})
+  .add({
+    targets: '.ml7 .letter',
+    translateY: ["1.1em", 0],
+    translateX: ["0.55em", 0],
+    translateZ: 0,
+    rotateZ: [180, 0],
+    duration: 750,
+    easing: "easeOutExpo",
+    delay: (el, i) => 50 * i
+  }).add({
+    targets: '.ml7',
+    opacity: 0,
+    duration: 200,
+    easing: "easeOutExpo",
+    delay: 5000
+  });
+
+    // Rotating Text
+    const rotatingText = document.querySelector('.rotating-text');
+    if (rotatingText) {
+        const textArray = [
+            'Frontend Developer',
+            'Backend Developer',
+            'Video Editor',
+            'Graphic Designer'
+        ];
+        let currentIndex = 0;
+
+        function changeText() {
+            const currentText = rotatingText.querySelector('.current');
+            const nextText = document.createElement('span');
+            nextText.className = 'next';
+            nextText.textContent = textArray[currentIndex];
+            rotatingText.appendChild(nextText);
+
+            setTimeout(() => {
+                if (currentText) {
+                    currentText.classList.remove('current');
+                    rotatingText.removeChild(currentText);
+                }
+                nextText.classList.remove('next');
+                nextText.classList.add('current');
+            }, 1000); // Tiempo de transición de texto
+
+            currentIndex = (currentIndex + 1) % textArray.length;
+        }
+
+        setInterval(changeText, 2500); // Intervalo de cambio de texto
+    }
+
+    let currentSlide = 0;

```

Comment:
I see you are using javascript to build your portfolio so I will leave comments based on this. Best practice was using Html and css to get acquainted with using them within a project when called for. 
To prevent global scope pollution, it would be nice to encapsulate variables within their respective functions. An example could be encapsulating "currentSlide"  and  "skillSlideIndex" within their own functions.